blocks/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Block(models.Model):
+    '''
+    This represents a block in the blockchain
+    '''
+    hash = models.CharField(max_length=100)
+    merkle_hash = models.CharField(max_length=100)
+    prev_block = models.OneToOneField('self', null=True, blank=True, on_delete=models.CASCADE)
+    timestamp = models.DateTimeField()
```

Comment:
Timestamp can be added as auto_now_add=True which will produce the default value of creation. Also, make it editable=False.

Reply:
done